tui_main.c

```diff
@@ -1903,6 +1908,17 @@ int main(int argc, char** argv) {
         exit(1);
       }
     } break;
+    case Argopt_bpm: {
+      init_bpm = atoi(optarg);
+      if (init_bpm < 0 ||
+          (init_bpm == 0 && strcmp(optarg, "0"))) {
```

Comment:
No need to break this across two lines -- doesn't exceed 80 columns. See the .clang-format file for the formatting rules, or just run `clang-format` on the file to get the proper output.